azure-functions-maven-plugin/src/it/http-trigger/src/main/java/com/microsoft/azure/Function.java

```diff
@@ -7,8 +7,8 @@
 package com.microsoft.azure;
 
 import java.util.*;
-import com.microsoft.azure.serverless.functions.annotation.*;
-import com.microsoft.azure.serverless.functions.*;
+import com.microsoft.azure.functions.annotation.*;
+import com.microsoft.azure.functions.*;
```

Comment:
Please rename function from hello to HttpTriggerJava similar to  [HttpTriggerCSharp.cs#L25](https://github.com/Azure/azure-functions-templates/blob/dev/Functions.Templates/Templates/HttpTrigger-CSharp/HttpTriggerCSharp.cs#L25)

Reply:
@pragnagopa @JunyiYi Done.